README.md

```diff
@@ -316,7 +296,7 @@ We'd love to highlight cool open-source projects that use this SDK! If you'd lik
 [dw-answers]: https://developer.ibm.com/answers/questions/ask/?topics=watson
 [stack-overflow]: http://stackoverflow.com/questions/ask?tags=ibm-watson
 
-[discovery]: https://www.ibm.com/watson/developercloud/discovery/api/v1/
+[discovery]: https://www.ibm.com/cloud/watson-discovery
```

Comment:
This link definition is never used. And the link that is used is wrong at line 38